examples/step-10/doc/results.dox

```diff
@@ -8,54 +8,58 @@ graphics. They are generated in Gnuplot format, and can be viewed with
-right now.
-
-
-
-The following table shows the triangulated computational domain for
-Q1, Q2, and Q3 mappings, for the original coarse grid (left), and a
-once uniformly refined grid (right). If your browser does not display
-these pictures in acceptable quality, view them one by one.
-
-<table style="width:80%" align="center">
-  <tr>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-10.ball_mapping_q1_ref0.png" alt=""></td>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-10.ball_mapping_q1_ref1.png" alt=""></td>
-  </tr>
-
-  <tr>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-10.ball_mapping_q2_ref0.png" alt=""></td>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-10.ball_mapping_q2_ref1.png" alt=""></td>
-  </tr>
-
-  <tr>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-10.ball_mapping_q3_ref0.png" alt=""></td>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-10.ball_mapping_q3_ref1.png" alt=""></td>
-  </tr>
-</table>
-
-These pictures show the obvious advantage of higher order mappings:
-they approximate the true boundary quite well also on rather coarse
-meshes. To demonstrate this a little further, the following table
-shows the upper right quarter of the circle of the coarse mesh, and
-with dashed lines the exact circle:
-
-<table style="width:80%" align="center">
-  <tr>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-10.quarter-q1.png" alt=""></td>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-10.quarter-q2.png" alt=""></td>
-    <td><img src="https://www.dealii.org/images/steps/developer/step-10.quarter-q3.png" alt=""></td>
-  </tr>
-</table>
+right now. Similarly, the third and fourth disable tic marks.
+
+The following table shows the triangulated computational domain for $Q_1$,
+$Q_2$, and $Q_3$ mappings, for the original coarse grid (left), and a once
+uniformly refined grid (right).
+
+<div class="twocolumn" style="width: 80%">
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_0_q1.svg">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_1_q1.svg">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_0_q2.svg">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_1_q2.svg">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_0_q3.svg">
+  </div>
+  <div>
+    <img src="https://www.dealii.org/images/steps/developer/step_10_ball_1_q3.svg">
+  </div>
+</div>
+
+These pictures show the obvious advantage of higher order mappings: they
+approximate the true boundary quite well also on rather coarse meshes. To
+demonstrate this a little further, here is part of the upper right quarter
+circle of the coarse meshes with Q2 and Q3 mappings, where the dashed red line
```

Comment:
Please use `$Q_2$ and $Q_3$, like you did above.

Reply:
Whoops. Thanks for catching that.